pimcore/modules/admin/controllers/MiscController.php

```diff
@@ -484,19 +484,26 @@ public function getAvailableActionsAction()
         $controller = $this->getParam("controllerName");
         $controllerClass = str_replace("-", " ", $controller);
         $controllerClass = str_replace(" ", "", ucwords($controllerClass));
+        $reflectionClass = "{$controllerClass}Controller";
         $controllerClass = preg_replace_callback("/([_])([a-z])/i", function ($matches) {
             return "/" . strtoupper($matches[2]);
         }, $controllerClass);
 
         $controllerDir = $this->getControllerDir();
         $controllerFile = $controllerDir . $controllerClass . "Controller.php";
         if (is_file($controllerFile)) {
-            preg_match_all("/function[ ]+([a-zA-Z0-9]+)Action/i", file_get_contents($controllerFile), $matches);
-            foreach ($matches[1] as $match) {
-                $dat = [];
-                $dat["name"] = strtolower(preg_replace("/[A-Z]/", "-\\0", $match));
-                $actions[] = $dat;
-            }
+            require_once $controllerFile;
+            $oReflectionClass = new \ReflectionClass($reflectionClass);
+            $methods = $oReflectionClass->getMethods(\ReflectionMethod::IS_PUBLIC);
+            $methods = array_filter(
+                $methods, function (\ReflectionMethod $method) {
+                return preg_match('/^([a-zA-Z0-9]+)Action$/', $method->getName());
+            });
+            $actions = array_values(array_map(
+                function (\ReflectionMethod $method) {
+	                return ["name" => preg_replace('/Action$/', '', $method->getName())];
```

Comment:
I think the following functionality is missing here, right? 
`$dat["name"] = strtolower(preg_replace("/[A-Z]/", "-\\0", $match));`

This transforms eg. `fooBar` to `foo-bar` which is the correct format for actions. 
Can you update your PR accordingly? Thanks!